spotify/templates/spotify/error.html

```diff
@@ -14,7 +14,7 @@
     </head>
     <body>
         {% include 'navbar.html' %}
-        <div class="container">
+        <div class="container-fluid">
```

Comment:
using bootstrap fluid container for better responsive layout. other templates already have this structure but i forgot about this particular page